src/renderer/utils/file-monitor.test.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable jest/prefer-expect-assertions */
+import fs from 'fs';
+
+import { FileMonitor } from './file-monitor';
+let a: [Event, string][];
+describe('file-monitor', () => {
+  it('deleting files should be reported', () => {
+    const fileMonitor = new FileMonitor(__dirname);
+    fileMonitor.watch(['unlink']);
+    fs.writeFileSync('test.txt', 'this is created for filemonitor test');
+    fs.unlinkSync('test.txt');
+    fileMonitor.getSubject().subscribe({
```

Comment:
Why does the subscription not happen earlier, i.e. before the file was unlinked?